### PR TITLE
Move More Collectors

### PR DESCRIPTION
**Pull request by SuperQ**
* Move scrapeEngineTokudbStatus to collector package.
* Move scrapeBinlogSize to collector package.
* Move scrapeSlaveStatus to collector package.
* Move scrapeGlobalVariables to collector package.

### CODE REVIEW

File: collector/engine_tokudb_test.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import (
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/smartystreets/goconvey/convey"
+	"gopkg.in/DATA-DOG/go-sqlmock.v1"
+)
+
+func Test_sanitizeTokudbMetric(t *testing.T) {
```
Comment (juliusv) [inline collector/engine_tokudb_test.go]: Do you want to remove the underscore here and below while you're at it, to make it more Go-idiomatic?

**Comment by juliusv** [comment]
:+1: otherwise

File: collector/engine_tokudb_test.go (hunk 1 above)
Comment (SuperQ) [inline collector/engine_tokudb_test.go]: Yea, I can go remove all those `_`

**Comment by juliusv** [comment]
:+1:

**Pull request merged**